documentation/asciidoc/accessories/iqaudio/getting_started.adoc

```diff
@@ -113,7 +113,7 @@ Enter the following to create a crontab daemon that will automatically start the
 $ crontab -e
 ----
 
-You will be asked to select an editor, we recommend you use "nano". Select it by entering the corresponding number, and press Enter to continue. The following line should be added to the bottom of the file:
+You will be asked to select an editor; we recommend you use Nano. Select it by entering the corresponding number, and press Enter to continue. The following line should be added to the bottom of the file:
```

Comment:
Nano here should probably remain lower-cased, as it's the name of a command?

Reply:
> Nano here should probably remain lower-cased, as it's the name of a command?

And be enclosed in backticks.

Reply:
Fixed